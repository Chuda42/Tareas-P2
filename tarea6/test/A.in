asociarEnMap 2 2.0
valorEnMap 2
insertarEnCP 8 8.0
prioritario
hacerVecinos 1 2 1.0
sonVecinos 1 2
sonVecinos 3 4
Fin
