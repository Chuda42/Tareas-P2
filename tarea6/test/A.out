1>Establecida la asociación.
2>2.00
3>Se insertó el elemento.
4>8
5>Nuevos vecinos: 1 2 1.00.
6>Son vecinos.
7>NO son vecinos.
8>Fin.
